src/Hyperion.Tests/Hyperion.Tests.csproj

```diff
@@ -2,7 +2,11 @@
   <Import Project="..\common.props" />
 
   <PropertyGroup>
+    <OutputType>Exe</OutputType>
     <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
+    <LangVersion>latest</LangVersion>
+    <StartupObject>Hyperion.Tests.Generator.Program</StartupObject>
```

Comment:
Fine with this - but I'll need to check the test output real fast to make sure everything is still there (pretty sure this PR would have exploded if it wasn't.)